cmd/node/node.go

```diff
@@ -575,7 +580,18 @@ func (app *SpacemeshApp) initServices(nodeID types.NodeID,
 			app.Config.HareEligibility.EpochOffset, app.Config.BaseConfig.LayersPerEpoch)
 	}
 
-	syncer := sync.NewSync(swarm, msh, app.txPool, atxdb, eValidator, poetDb, syncConf, clock, app.addLogger(SyncLogger, lg))
+	bCfg := blocks.Config{
+		Depth:       app.Config.Hdist,
+		GoldenATXID: goldenATXID,
+	}
+	blockListener := blocks.NewBlockHandler(bCfg, msh, eValidator, lg)
+
+	fetcher := fetch.NewFetch(app.Config.FETCH, swarm, app.addLogger(Fetcher, lg))
```

Comment:
nameing is kinda weird here and also in general,  - you are using `fetch` and `fetcher` alternately in a way that doesn't  really make sense.